app/views/merchant/items/index.html.erb

```diff
@@ -2,4 +2,9 @@
 
 <% @items.each do |item| %>
```

Comment:
We'll want to add formatting to this so it displays better but we can do that in a refactor.